---
title: "Hyun-Hwan Jeong's CV"
author: Hyun-Hwan Jeong
date: "`r Sys.Date()`"
output:
  pagedown::html_resume:
    # set it to true for a self-contained HTML page but it'll take longer to render
    self_contained: false
---

Aside
================================================================================


![Hyun-Hwan Jeong](./profile.jpg){width=80%}


Contact Info {#contact}
--------------------------------------------------------------------------------

- <i class="fa fa-envelope"></i> hyun-hwan.jeong@bcm.edu
- <i class="fa fa-github"></i> [github.com/hyunhwaj](https://github.com/hyunhwaj)
- For more information, please contact me via email.


Skills {#skills}
--------------------------------------------------------------------------------

- Experienced in algorithm design and statistical learning models.

- Full experience with next generation sequencing data analysis.

- Highly skilled in R and Python.


Disclaimer {#disclaimer}
--------------------------------------------------------------------------------

This resume was made with the R package [**pagedown**](https://github.com/rstudio/pagedown).

Last updated on `r Sys.Date()`.



Main
================================================================================

Hyun-Hwan Jeong {#title}
--------------------------------------------------------------------------------

### Postdoctoral Associate

I am a postdoctoral associate in the [Department of Molecular and Human Genetics](https://www.bcm.edu/departments/molecular-and-human-genetics) at the [Baylor College of Medicine](https://www.bcm.edu/) and [Jan and Dan Duncan Neurological Research Institute](https://nri.texaschildrens.org/). I have received my Ph.D. degree in computer science from [Ajou University](https://www.ajou.ac.kr/en/) in August 2015 and started the postdoctoral position in the lab of [Dr. Zhandong Liu](http://www.liuzlab.org/), and co-mentored by [Dr. Huda Zoghbi](https://en.wikipedia.org/wiki/Huda_Zoghbi), since September 2015.

During my Ph.D. studies, I was highly focused on the development of integrative network analysis framework for multiple omics data using information-theoretic measure . I am now focusing on developing a quantification algorithm and computational pipeline construction for NGS data for the neurological disorder.

I am also a developer of [SalmonTE](https://github.com/LiuzLab/SalmonTE) and [CB<sup>2</sup>](https://github.com/LiuzLab/CB2).




Education {data-icon=graduation-cap data-concise=true}
--------------------------------------------------------------------------------

### Ajou University

Ph.D. in Computer Science and Engineering

Suwon, South Korea

2009 - 2015

Thesis: Integrative network analysis framework for multiple omics data using information-theoretic measure

### Ajou University

M.Eng. in Information and Communication Technology

Suwon, South Korea

2007 - 2009

Thesis: Missing SNP genotype imputation by heuristic algorithm

### Ajou University

B.S. in Information and Computer Engineering

Suwon, South Korea

2003 - 2007

Research Experience {data-icon=laptop}
--------------------------------------------------------------------------------

### Postdoctoral Associate

Department of molecular and human genetics, Baylor college of medicine

Houston, Texas

2015 - Present

- Mentor: Huda Zoghbi and Zhandong Liu

### Research assistant

Health Avatar Project, National Core Research Center(NCRC)

Suwon, South Korea

2014 - 2015

- Developed an multi-omics data intergration software (MINA) integrative omics data analysis using clinical outcome-guided mutual information network.


Teaching Experience {data-icon=chalkboard-teacher}
--------------------------------------------------------------------------------

### C Programming

Part time Instructor of C programming at Ajou University. 

Suwon, South Korea

2014 - 2015

### Software problem solving skill training

Instructor of the training course at Ajou University.

Suwon, South Korea

2015


Selected Peer-Reviewed Publications {data-icon=file}
--------------------------------------------------------------------------------

```{r, results='asis', echo=FALSE}
suppressMessages(library(glue))
suppressMessages(library(tidyverse))
suppressMessages(library(stringr))
suppressMessages(library(rcrossref))

suppressMessages(doi <- read_csv("doi.csv"))

for(d in doi$doi) {
  cn_obj <- cr_cn(dois = d, "citeproc-json")
  
  title <- cn_obj$title
  journal <- cn_obj$`container-title`
  year <- cn_obj$`published-print`$`date-parts`[1]
  if(is.null(year)) year <- cn_obj$issued$`date-parts`[1]
  
  link <- cn_obj$URL
  
  clean_author_list <- function(cn_obj) {
    author <- cn_obj$author %>% unite("name", c("given", "family"), sep = " ")
    author <- author$name
    mine <- author %in% c("Hyun-Hwan Jeong", "Hyun-hwan Jeong")
    author[mine] <- str_c("**", author[mine], "**")
    str_c(author, collapse = ", ")
  }
  
  author <- clean_author_list(cn_obj)
  
  cat("### [{title}]({link})" %>% glue)
  cat('\n\n')
  cat('{journal}' %>% glue)
  cat('\n\n')
  cat('N/A') 
  cat('\n\n')
  cat('{year}' %>% glue) 
  cat('\n\n')
  cat('{author}' %>% glue) 
  cat('\n\n')
}

```